docker/dev_entrypoint.sh

```diff
@@ -20,6 +20,7 @@ TARGET_GAS_LIMIT=${TARGET_GAS_LIMIT:-4294967295}
   --rpc \
   --rpcaddr $HOSTNAME \
   --rpcvhosts='*' \
+  --rpccorsdomain='*' \
```

Comment:
Good catch updating the `dev_entrypoint.sh` as well based on changes here: https://github.com/ethereum-optimism/go-ethereum/commit/493fe631f78d6d1463b5727fd828a98c29efb550#diff-1fd0da033753906a29729c3b71ee9e6fR26